src/Microsoft.Diagnostics.Runtime/src/DataReaders/Minidump/MinidumpReader.cs

```diff
@@ -38,7 +36,10 @@ public MinidumpReader(string crashDump, Stream stream)
 
             DisplayName = crashDump;
 
-            _minidump = new Minidump(crashDump, stream);
+            stream.Dispose();
```

Comment:
Is it safe here to dispose of a stream you don't own? Probably better to leave the disposing to the caller.

Reply:
See the second commit. :)

Reply:
No worries. The caller, `DataTarget`, was designed to transfer the ownership of the stream to the data reader. CA2000 doesn't know that either so we silenced it.

https://github.com/microsoft/clrmd/blob/ea1d957850c92a4f9a2ac73b5b72517bb83be0c6/src/Microsoft.Diagnostics.Runtime/src/DataTargets/DataTarget.cs#L245